lib/dhis2/api/analytic.rb

```diff
@@ -0,0 +1,16 @@
+module Dhis2
+  module Api
+    class Analytic < Base
+      class << self
+        def list(client, periods:, organisation_unit:, data_elements:)
```

Comment:
use plural for organisation units - the general case is there may be several of them